### DISCUSSION TITLE
Meeting Notes #1 - Dec 30, 2023

### DISCUSSION THREAD

**Discussion by jiangbonadia** [segmentfault/dev-together#3] [Meeting Notes]
Time: December 30, 2023, 22:00 (GMT+8) 
Link: https://meeting.tencent.com/dm/MRqu5vbhTRvR
Attendees: 江波（Nadia）、王尚、林旅强（Richard）、Kate Shao、袁滚滚、庄表伟、许银、曾辉、邢云（Cici）

## Agenda
1. Agenda 1：江波介绍 Dev.Together 会议背景及开放协作方式
2. Agenda 2：讨论 24 年会议的内容（大家期待的 Track 方向）
3. Agenda 3：筹备组分工（内容组+传播组），会务工作暂时仍继续交由 SegmentFault team 负责

### 1. 目标人群

- Kate：是否要破圈？
- Richard：应该适度破圈
- 波波：适度破圈，可以先破圈到已经在做 DevRel 但没有意识到自己在做 DevRel 的人

### 2. 会议分工讨论

- 会议会务服务工作仍交由 SegmentFault team 负责
- 设立会议内容工作组
	- 核心筹备组牵头决定 Track 的设置及发起 CFP
	- 核心筹备组召集/邀请 Track Chair 进行议题审核
- 设立会议传播工作组
	- 传播内容组：筹备传播内容（如：文章、海报、播客、直播、短视频等）
		- 王尚报名播客组工作
	- 传播渠道组：社区合作、内容分发
- **征集会议主席 4 名，工作组组长若干，欢迎 WeChat Group 群内/私聊波波报名，或直接到会议纪要下回帖**

![image](https://github.com/segmentfault/dev-together/assets/1227386/053a4177-8a17-472a-9c60-8ed38bc799e2)

### 3. 会议内容讨论

* Track 划分方式
	- **划分方式**：技术领域、从业时长、工作/技能方向（同往年）
	- **初步结论**：按照工作/技能方向（同往年）划分会场，但增加 tag 的分类方式，即为每个 Topic 打上 tag（如技术领域、适合 junior/senior 受众），用户除了可以按照会场检索内容，也可以按照标签检索自己感兴趣的内容

* 大家感兴趣的内容方向
  - 社区
  	- 社区运营（中立行业社区、媒体社区、甲方社区）- 王尚
  	- 社区治理（规则）- 庄表伟
  - 同业竞争/竞合 - Kate Shao
  - AI 开发者生态 - 波波、滚滚、Richard
  - Web3 开发者生态 - 许银
  - **更多话题将在群里发起征集，请大家在 WeChat Group 中回复，或回复本帖**，收集完毕后会在各个 DevRel 微信群中 vote

## To-Dos
1. 会议筹备组分工征集
2. 会议内容筹备组内征集 + 开放投票

**Comment by HungryFish034** [comment]
李梦 报名传播内容组-文案组 🙋

**Comment by Nliver** [comment]
加入支持！

**Reply by jiangbonadia** [reply]
小明是否考虑勇担传播重任？来做传播渠道组的组长和大会传播方向的 co-chair？

**Comment by shinyzhu** [comment]
报名一起来办DevRel～
个人便好“内容”方面。

**Reply by richardlinq** [reply]
@shinyzhu 怎么样的“内容”，能否举例哈

**Comment by LinuxSuRen** [comment]
报名播客工作
另外，对纯社区化运作（背后没有商业公司左右）比较感兴趣，看是否可以成为一个 track，邀请相关社区来分享经验。

**Reply by jiangbonadia** [reply]
请问有拟邀的社区清单或者举例吗？比如 ALC Beijing、开源社？不知道我理解得对不对

**Reply by LinuxSuRen** [reply]
确实是这类社区，名单还没想好

**Comment by WayneZhon** [comment]
钟伟纯报名渠道传播组（希望思否给予相关合作单位以及参会人员报名的时候提供相关曝光渠道），还有就是金蝶这边有多功能厅，能容纳两三百人，不过就是一个投影仪没有其他太多的东西，看需不需要哈，另外希望增加以下讨论几个话题：
1、如何推动平台的技术老师对外布道
2、如何做好传递平台功能收集用户反馈的同时关注开发者的成长
3、如何做好运营驱动产品改造建设

欢迎交流~

**Reply by LinuxSuRen** [reply]
想请教下，“平台的技术老师”指的是？

**Reply by WayneZhon** [reply]
平台技术老师，指的是我们产品平台部的一些技术大牛，本质上他们是最了解平台的，但是很多人都认为我做好产品就行了，很少人愿意对外布道，即便加入鲲鹏会、技术媒体也很少对外去布道

**Comment by WayneZhon** [comment]
您的邮件，我已收到，谢谢！

**Comment by Ning-xz** [comment]
内容方向希望可以增加【企业内部开发者生态运营】的话题讨论

**Reply by WayneZhon** [reply]
借此还可以增加一个开源话题，如何做好内部的开源建设然后走向外部开源，因为很多大公司都是先做好内部开源，形成一些资产复用后挑选部分进行对外开源

**Comment by WayneZhon** [comment]
想增加一个track：如何做好数字化运营？我们技术运营基本围绕传递产品价值、收集用户反馈推动产品迭代优化以及帮助用户成长项目成功，但这都离不开一个数字化的运营，标签体系，产品埋点等等，想听听实实在在的案例
